### DISCUSSION TITLE
Link GitHub to Rstudio

### DISCUSSION THREAD

**Discussion by 0abdullah0sami0** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#54] [Q&A] [answered]
After finishing a project in Rstudio, I publish it in a new GitHub repisotory using usethis::use_git(),usethis::use_github(), however, I can't keep updating it and making commits through Rstudio because I don't see the Git button in Rstudio anymore.

Link to my project:
https://github.com/0abdullah0sami0/Insurance-Analysis

**Comment by BatoolMM** [comment, marked as answer]
Salam @0abdullah0sami0, 

Thank you for submitting your question. Did you try to restart RStudio and can you include a screenshot?

**Reply by 0abdullah0sami0** [reply]
Thank you @BatoolMM for your answer. I managed to solve the problem eventually after updating my R packages.

**Reply by BatoolMM** [reply]
ممتاز جدا 😊
